### ISSUE TITLE
Quick reply buttons do not move the user's focus to the start of the message response from the chatbot.

### ISSUE THREAD

**Issue by cryptasian** [microsoft/BotFramework-WebChat#4992]
### Is it an issue related to Adaptive Cards?

No.

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Focus management, Live region

### What is the public URL for the website?

https://edfinancial.com

### How to reproduce the issue?

This is feedback from the accessibility company Allyant who has been testing our website and has people who use screenreaders and other assistive technologies as a part of their testing team.

![new-message-focus](https://github.com/microsoft/BotFramework-WebChat/assets/30207740/6490d8da-094f-402b-bf02-c45a15816154)

### What do you expect?

When a button option is selected, the focus should be on the first message that was sent by the chatbot in response to the button selection.

Set focus to the new message displayed in the chat window when selecting a button by using the JavaScript focus() method to move keyboard focus to the target element when the button is activated. Ensure that the target element can receive keyboard focus by using a natively-focusable element or giving the element a tabindex="-1" attribute.

### What actually happened?

When a button option is selected, the focus does not go to the first message that was sent by the chatbot in response to the button selection.

### WCAG reference

https://www.w3.org/WAI/WCAG22/Understanding/focus-order

**Comment by compulim** [comment]
Suggested actions are predefined messages and they are designed to help end-user quickly type out the message. They are designed as part of the send box and is not part of the message.

Thus, when the suggested action is sent and disappear from the screen, the focus will go to the send box, which is the container of the suggested actions.
